config/generic-steval-3dp001v1.cfg

```diff
@@ -0,0 +1,327 @@
+# This file contains common pin mappings for STEVAL-3DP001V1 board.
+# to use this config, the firmware should be compiled for the STM32F4
+
+# See docs/Config_Reference.md for a description of parameters.
+# Refer to https://www.st.com/en/motor-drivers/l6474.html for more info about the L6474 stepper driver
+
+
+[stepper_x]
+step_pin: PE14
+dir_pin: !PE15
+microsteps: 16
+rotation_distance: 40
+endstop_pin: ~!PD8
+position_endstop: 0
+position_max: 300
+homing_speed: 50
+
+[stepper_y]
+step_pin: PB10
+dir_pin: !PE9
+microsteps: 16
+rotation_distance: 40
+endstop_pin: ~!PD9
+position_endstop: 0
+position_max: 255
+homing_speed: 50
+
+[stepper_z]
+step_pin: PC6
+dir_pin: PC0
+microsteps: 16
+rotation_distance: 8
+endstop_pin: ~!PD10
+position_endstop: 0
+position_min: 0
+position_max: 300
+
+[extruder]
+step_pin: PD12
+dir_pin: PC13
+microsteps: 16
+rotation_distance: 7.921
+nozzle_diameter: 0.400
+filament_diameter: 1.750
+heater_pin: PC7
+sensor_type: EPCOS 100K B57560G104F
+pullup_resistor: 1000  # for steval board 1k pull up resistor is used
+sensor_pin: PA0
+control: pid
+pid_Kp: 24.575
+pid_Ki: 1.085
+pid_Kd: 139.157
+min_temp: 0
+max_temp: 265
+
+[heater_bed]
+heater_pin: PD14
+sensor_type: EPCOS 100K B57560G104F
+pullup_resistor: 1000  # for steval board 1k pull up resistor is used
+sensor_pin: PC2
+control: pid
+pid_Kp: 71.255
+pid_Ki: 0.956
+pid_Kd: 1328.008
+min_temp: 0
+max_temp: 95
+
+#layer fan
+[fan]
+pin: PE8
+
+[heater_fan extruder]
+pin: PC4
+heater : extruder
+
+[mcu]
+serial: /dev/serial/by-id/usb-STMicroelectronics_STM32_STLink_0674FF514852897267245621-if02
+baud: 115200
```

Comment:
You have stated a custom speed, we generally do not set this unless their is a very good reason to do so, 250000 default works on the majority of known boards. Have you tested that it will not work at the default speed?